tensorflow/compiler/xla/tests/convolution_test.cc

```diff
@@ -1947,7 +1946,7 @@ XLA_TEST_F(ConvolutionTest, ConvolveF32BackwardInputGroupedConvolution) {
 
 class ConvolutionHloTest : public HloTestBase {};
 
-// double datatype is not yet supported in ROCm  
+// double datatype is not yet supported in ROCm
 XLA_TEST_F(ConvolutionHloTest, DISABLED_ON_GPU_ROCM(DISABLED_ON_CPU(ConvolveF64Forward))) {
```

Comment:
@deven-amd can we remove `DISABLED_ON_CPU`.

Reply:
@whchung 

can I push that change out in a different PR? 

This one has passed through CI already, and I anticipate filing a few more PRs this week :)